backend/app/tests/conftest.py

```diff
@@ -60,3 +60,8 @@ def pytest_runtest_setup(item):
         pytest.skip("This test is destructive and the target URL is"
                     "considered a sensitive environment. If this test"
                     "is not destructive add the 'nondestructive' marker.")
+
+
+@pytest.fixture(scope="session")
```

Comment:
The integration test errors happen because you can't resolve `http://frontend/api`. That will result in a 404. You'll need to adjust this fixture to switch to "http://backend/api" if the `base_url` is "http://frontend".